src/Components/Positions/index.tsx

```diff
@@ -328,13 +328,13 @@ export const Positions = ({ setOutputTokenAddressToBuy, tokenOutMaturity }: { se
               <th>Position</th>
               <th className='no-wrap'>
                 Net Value
-                <Text
+                {wrapToNativeAddress(positions?.[0]?.pool?.TOKEN_R) ? <Text
                   className='text-link'
                   onClick={() => {
                     setValueInUsdStatus(valueInUsdStatus === VALUE_IN_USD_STATUS.USD ? VALUE_IN_USD_STATUS.TOKEN_R : VALUE_IN_USD_STATUS.USD)
                   }}
-                >{valueInUsdStatus === VALUE_IN_USD_STATUS.USD ? ` ⇄ ${tokens[wrapToNativeAddress(positions?.[0].pool.TOKEN_R)]?.symbol}` : ' ⇄ USD'}
-                </Text>
+                >{valueInUsdStatus === VALUE_IN_USD_STATUS.USD ? ` ⇄ ${tokens[wrapToNativeAddress(positions?.[0]?.pool?.TOKEN_R)]?.symbol}` : ' ⇄ USD'}
```

Comment:
same comment above

Reply:
Comment above should be modified to read "positions.length > 0" in order to make the code functional.

Reply:
why? what's wrong with `?.length`?

Reply:
Sorry i miss `?.` in my comment, nothing wrong with use `?.length`